test/test_wit.py

```diff
@@ -7,37 +7,49 @@ class TestWit(unittest.TestCase):
     def setUp(self):
         pass
 
+    def test_vul_leggerwaarden(self):
+        """test create values of theoretical profiel"""
+        pass
+
     def test_create_leggerpunten(self):
         """test create points of theoretical profiel"""
         
         line = TLine([(0, 0), (0, 10)])
-        TI_waterbr = 10
-        TI_talulbr = 3
-        TI_bodembr = 5
+        line_id = 1
+        name = 'test lijn'
+        ti_waterbr = 10
+        ti_talulbr = 3
+        ti_knkbodr = 8
 
-        profiel_dict = create_leggerpunten(line,TI_waterbr, TI_talulbr, TI_bodembr)
+        profiel_dict = create_leggerpunten(line, line_id, name, ti_waterbr, ti_talulbr, ti_knkbodr)
```

Comment:
Nee, omdat ik dus problemen heb met de assertion op de dicts, omdat de volgorde van de dict in het resultaat 'willekeurig' lijkt. Aangepaste versie net geprobeerd te committen maar krijg error by synchronisaties :(